### PR TITLE
Manage tokens in InfluxDB Cloud Serverless

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-serverless/admin/tokens/create-token.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+## Create a token using the InfluxDB API
+
+Use the `/api/v2/authorizations` InfluxDB API endpoint to create a token.
+
+{{< api-endpoint method="POST" endpoint="http://localhost:8086/api/v2/authorizations" api-ref="/influxdb/v2.7/api/#operation/PostAuthorizations" >}}
+
+Include the following in your request:
+
+- **Headers**
+  - **Authorization**: `Bearer API_TOKEN`
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/create-token.md:190]: Does `Bearer` work in Serverless?

File: content/influxdb/cloud-serverless/admin/tokens/delete-token.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```sh
+# Delete the first authorization listed for the user.
+curl --request GET \
+  "https://cloud2.influxdata.com/api/v2/authorizations?user=user2" \
+  --header "Authorization: Bearer API_TOKEN" \
+  --header 'Content-type: application/json' \
+| jq .authorizations[0].id \
+| xargs -I authid curl --request DELETE \
+  https://cloud2.influxdata.com/api/v2/authorizations/authid \
+  --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/delete-token.md:92]: Same

File: content/influxdb/cloud-serverless/admin/tokens/update-tokens.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+Use the `/api/v2/authorizations` InfluxDB API endpoint to update the description
+and status of a token.
+
+{{< api-endpoint method="PATCH" endpoint="https://cloud2.influxdata.com/api/v2/authorizations/{AUTH_ID}" api-ref="/influxdb/v2.7/api/#operation/PatchAuthorizationsID" >}}
+
+Include the following in your request:
+
+- **Headers**:
+  - **Authorization**: `Bearer API_TOKEN`
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/update-tokens.md:114]: Same

File: content/influxdb/cloud-serverless/admin/tokens/update-tokens.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+# Update the description and status of the first authorization listed for the user.
+
+curl --request GET \
+  "https://cloud2.influxdata.com/api/v2/authorizations?user=user2" \
+  --header "Authorization: Bearer API_TOKEN" \
+  --header 'Content-type: application/json' \
+| jq .authorizations[0].id \
+| xargs -I authid curl --request PATCH \
+  https://cloud2.influxdata.com/api/v2/authorizations/authid \
+  --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/update-tokens.md:136]: Same

File: content/influxdb/cloud-serverless/admin/tokens/update-tokens.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  to update
+
+### Disable a token
+
+{{% code-placeholders "API_TOKEN" %}}
+```sh
+# Update the description and status of the first authorization listed for the user.
+
+curl --request GET \
+  "https://cloud2.influxdata.com/api/v2/authorizations?user=user2" \
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/update-tokens.md:130]: Are `"` necessary if we're not quoting below?

File: content/influxdb/cloud-serverless/admin/tokens/view-tokens.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    name: View tokens
+    parent: Manage tokens
+weight: 202
+---
+
+View API tokens and permissions using the InfluxDB user interface (UI),
+the `influx` command line interface (CLI), or the InfluxDB API.
+
+{{% note %}}
+To follow best practices for secure API token generation and retrieval, InfluxDB Cloud enforces access restrictions on API tokens.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/view-tokens.md:16]:
```suggestion
To follow best practices for secure API token generation and retrieval, {{% cloud-name %}} enforces access restrictions on API tokens.
```

File: content/influxdb/cloud-serverless/admin/tokens/view-tokens.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+  - InfluxDB Cloud UI only allows access to the API token value immediately after the token is created.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/view-tokens.md:17]:
```suggestion
  - InfluxDB UI only allows access to the API token value immediately after the token is created.
```

File: content/influxdb/cloud-serverless/admin/tokens/view-tokens.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+{{< api-endpoint method="GET" endpoint="https://cloud2.influxdata.com/api/v2/authorizations" api-ref="/influxdb/cloud-serverless/api/#operation/GetAuthorizations" >}}
+
+- [View a single token](#view-a-single-token)
+- [Filter the token list](#filter-the-token-list)
+
+Include the following in your request:
+
+- **Headers**:
+  - **Authorization**: `Bearer API_TOKEN`
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/tokens/view-tokens.md:88]: Same

File: shared/text/api/cloud-serverless/auth/token-create.sh [hunk 9]
```diff
@@ -0,0 +1,34 @@
+curl -v --request POST \
+  https://cloud2.influxdata.com/api/v2/authorizations \
+  --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline shared/text/api/cloud-serverless/auth/token-create.sh:3]: Same

File: shared/text/api/cloud-serverless/auth/tokens-create-with-user.sh [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+function create_token_with_user() {
+  curl --request POST \
+    "https://cloud2.influxdata.com/api/v2/users/" \
+    --header "Authorization: Bearer API_TOKEN" \
+    --header 'Content-type: application/json' \
+    --data "{\"name\": \"$1\"}"
+  
+  curl --request GET \
+    "https://cloud2.influxdata.com/api/v2/users?name=$1" \
+    --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline shared/text/api/cloud-serverless/auth/tokens-create-with-user.sh:21]: Same

File: shared/text/api/cloud-serverless/auth/tokens-create-with-user.sh [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+        "userID": .id,
+        "description": $USER,
+        "permissions": [
+           {"action": "read", "resource": {"type": "buckets"}}
+         ]
+      }' | \
+  
+  curl --request POST \
+    "https://cloud2.influxdata.com/api/v2/authorizations" \
+    --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline shared/text/api/cloud-serverless/auth/tokens-create-with-user.sh:36]: Same

File: shared/text/api/cloud-serverless/auth/tokens-view-filter.sh [hunk 12]
```diff
@@ -0,0 +1,25 @@
+# The example below uses the common `curl` and `jq` command-line tools
+# with the InfluxDB API to do the following:
+# 1. Find a user by username and extract the user ID.
+# 2. Find the user's authorizations by user ID.
+# 3. Filter for `active` authorizations that have `write` permission.
+
+function list_write_auths() {
+  curl "https://cloud2.influxdata.com/api/v2/users/?name=$1" \
+    --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline shared/text/api/cloud-serverless/auth/tokens-view-filter.sh:9]: Same

File: shared/text/api/cloud-serverless/auth/tokens-view.sh [hunk 13]
```diff
@@ -0,0 +1,4 @@
+curl --request GET \
+	"https://cloud2.influxdata.com/api/v2/authorizations" \
+  --header "Authorization: Bearer API_TOKEN" \
```
Comment (jstirnaman) [inline shared/text/api/cloud-serverless/auth/tokens-view.sh:3]: Same

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-serverless/admin/tokens/update-tokens.md (hunk 5 above)
Comment (sanderson) [reply content/influxdb/cloud-serverless/admin/tokens/update-tokens.md:130]: No, they're not. I'll take them out (copypasta)

**Review by sanderson — COMMENTED** [review]
Double checking on `Bearer` support.

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**